tests/scripts/delete_all_indexes.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+import marqo
+
+
+def cleanup_documents_from_all_indices():
+    local_marqo_settings = {
+        "url": os.environ.get("MARQO_URL", 'http://localhost:8882'),
+    }
+    api_key = os.environ.get("MARQO_API_KEY", None)
+    if api_key:
+        local_marqo_settings["api_key"] = api_key
+    client = marqo.Client(**local_marqo_settings)
+    indexes = client.get_indexes()
+    for index in indexes['results']:
+        if client.config.is_marqo_cloud:
+            if index.get_status()["index_status"] == marqo.enums.IndexStatus.CREATED:
```

Comment:
tests wouldn't finish if some index is creating

Reply:
otherwise we'd just have to wait and something isn't working correct, we could add few minutes waiting time for those that are creating and clear them after that